R/utils-classes.R

```diff
@@ -324,6 +325,7 @@ show_query.bcdc_sf <- function(x, ...) {
 
 }
 
+# collapse vector of cql statements into one
 finalize_cql <- function(x, con = cql_dummy_con) {
   if (is.null(x)) return(NULL)
   dbplyr::sql_vector(x, collapse = " AND ", con = con)
```

Comment:
I wonder if it is easier here to possibly just rely on `paste0` and avoid dealing with the dummy connection? Then we feed that to `dbplyr::sql`? Does that make this less fragile?

Reply:
Could do... although `sql_vector` does some other things to ensure the sql is valid like making sure escaping is done correctly... because we define the type of escaping that is required for this type of sql based on the dummy connection (e.g., [here](https://github.com/bcgov/bcdata/blob/10720c95eeb96b1a136768b2ebbad2f77eaf08eb/R/cql-translator.R#L114-L128). Granted, all that should have been taken care of beforehand in `cql_translate()`... Also if the dummy connection is fragile here it will be fragile everywhere!

Reply:
That's a good point! In the weeds here I think. I'd merge.